<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Ashwini" failures="1" tests="2" name="Default test(failed)" time="22.905" errors="0" timestamp="2022-11-09T10:04:04 IST">
  <testcase classname="TestCases.InventoryPageTest" name="selectBoxTest" time="7.572"/>
  <testcase classname="TestCases.InventoryPageTest" name="gotonextpageTest" time="1.1">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [DESCRIPTION1] but found [DESCRIPTION]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [DESCRIPTION1] but found [DESCRIPTION]
at TestCases.InventoryPageTest.gotonextpageTest(InventoryPageTest.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- gotonextpageTest -->
  <testcase classname="TestCases.InventoryPageTest" name="removeTest" time="0.0">
    <ignored/>
  </testcase> <!-- removeTest -->
</testsuite> <!-- Default test(failed) -->
